src/ui/router.go

```diff
@@ -122,6 +122,21 @@ func initRouters() {
 
 	beego.Router("/registryproxy/*", &controllers.RegistryProxy{}, "*:Handle")
 
+	//APIs for chart repository
+	chartRepositoryAPIType := &api.ChartRepositoryAPI{}
+	beego.Router("/api/chartserver/health", chartRepositoryAPIType, "get:GetHealthStatus")
+	beego.Router("/api/:repo/charts", chartRepositoryAPIType, "get:ListCharts")
```

Comment:
Suggest we add a prefix to all the chart related APIs, for example `/api/chart_registry/:repo/charts`

Reply:
Maybe we can enable context

Reply:
Will find a proper way and submit code in separate PR